scripts/prepdocs.ps1

```diff
@@ -52,12 +52,17 @@ if ($env:AZURE_TENANT_ID) {
   $tenantArg = "--tenantid $env:AZURE_TENANT_ID"
 }
 
+if ($env:USE_FEATURE_INT_VECTORIZATION) {
+  $integratedVectorizationArg = "--useIntVectorization $env:USE_FEATURE_INT_VECTORIZATION"
```

Comment:
Our other args are lowercase so I've been trying to keep that consistent (I changed the keyVault args in my PR). So it'd be "useintvectorization" or maybe just "intvectorization" for readability.